ee/clickhouse/migrations/0023_dead_letter_queue_tags.py

```diff
@@ -0,0 +1,14 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.dead_letter_queue import DEAD_LETTER_QUEUE_TABLE_MV_SQL, KAFKA_DEAD_LETTER_QUEUE_TABLE_SQL
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+operations = [
+    migrations.RunSQL(f"DROP TABLE events_dead_letter_queue_mv ON CLUSTER {CLICKHOUSE_CLUSTER}"),
+    migrations.RunSQL(f"DROP TABLE kafka_events_dead_letter_queue ON CLUSTER {CLICKHOUSE_CLUSTER}"),
```

Comment:
if someone was trying to write to either immediately after this it would just fail, but that would be fine, correct?

Reply:
yes that'd be fine. also note this is a normal rather than async migration.

Other times we've done this: 0009, 0012, 0013, 0020

Reply:
yup normal migrations run at the same time as we're using the Posthog app, was why I was clarifying just in case.